nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/configuration/RequestTracingServiceConfiguration.java

```diff
@@ -29,11 +30,12 @@ and Distribution License("CDDL") (collectively, the "License").  You
 public interface RequestTracingServiceConfiguration extends ConfigBeanProxy, ConfigExtension {
 
     @Attribute(defaultValue = "false", dataType = Boolean.class)
-    Boolean getEnabled();
+    String getEnabled();
     void enabled(String value) throws PropertyVetoException;
 
-    @Attribute(defaultValue = "3")
-    Long getThresholdValue();
+    @Attribute(defaultValue = "3", dataType = Long.class)
+    @Min(value = 0)
```

Comment:
@mulderbaba Same here, haven't spotted anything that indicates it can be less than 0 - unless I'm being blind to it anyway.